src/ci/docker/dist-x86_64-redox/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:16.04
```

Comment:
[This folder](https://github.com/rust-lang/rust/tree/master/src/ci/docker/scripts) for shared scripts.